src/ophys_etl/transforms/postprocess_rois.py

```diff
@@ -133,6 +139,10 @@ def binarize_and_create(self):
                 binarized_coo_rois, motion_border, movie_shape,
                 self.args['npixel_threshold'])
 
+        if self.args['morphological_ops']:
+            compatible_rois = [morphological_transform(roi, shape=movie_shape)
```

Comment:
I guess I'm curious, was there a specific reason the morphological transform had to happen after `coo_rois_to_lims_compatible` rather than after `binarize_roi_mask`? A lot of functions like `full_mask_constructer` and `roi_from_full_mask` would probably be way less unwieldy (maybe even unnecessary) if dealing with `coo_matrix` instead of the `DenseROI` representation?

Reply:
I think as long as the `morphological_transform` also returns a coo_matrix, you wouldn't even need the `roi_from_full_mask` function either since `coo_rois_to_lims_compatible` would take care of that.

See: https://github.com/AllenInstitute/ophys_etl_pipelines/blob/49a886c94652b7fd7533fafd04caf0a23190b34f/src/ophys_etl/transforms/roi_transforms.py#L210

Reply:
I agree, it could have gone higher up and taken advantage of the alternate representation. I was thinking about this more in terms of the output format, rather than the input format.Thus far, this repo only goes one way (coo -> lims). Since we're delivering lims format at the end of postprocess, I thought it opened up the following possibility with no required code that does not yet exist (i.e. lims -> coo):
* someone segments with the morphological filter off, and sees their ROIs.
* they can then pass that format ROI right through this function to do an A/B test
This is something I can anticipate being useful if this filter turns out not to be the final word on these ROIs.
More and more, as we add to `roi_transforms.py` I wonder why we don't have a consistent ROI class in this repo that handles both representations.